spec/spec_helper.rb

```diff
@@ -48,6 +48,29 @@
 GETRF_EXAMPLE_ARRAY = [-1,0,10,4,9,2,3,5,7,8,1,6]
 GETRF_SOLUTION_ARRAY = [9.0, 2.0, 3.0, 5.0, 7.0/9, 58.0/9, -4.0/3, 19.0/9, -1.0/9, 1.0/29, 301.0/29, 130.0/29]
 
+TAU_SOLUTION_ARRAY = [1.8571428571428572,1.9938461538461538, 0.0]
+
+GEQRF_SOLUTION_ARRAY =[                -14.0,                -21.0, 14.000000000000002,
+                         0.23076923076923078,  -175.00000000000003,  70.00000000000001,
+                        -0.15384615384615385, 0.055555555555555546,              -35.0]
+
+R_SOLUTION_ARRAY   = [-159.2388143638353, -41.00131005172065, -56.75123892439876,  -90.75048729628048, 
+                                     0.0, 25.137473501580676,  2.073591725046292,   9.790607357775713, 
+                                     0.0,                0.0, -20.83259700334131, -17.592414929551445]
+
+Q_SOLUTION_ARRAY_1 = [-0.632455532033676, -0.5209522876558295, -0.3984263084135902,  -0.41214704991068,
+                    -0.42783756578748666, -0.20837937347171134, 0.876505919951498, 0.07259770177184455,
+                    -0.48364246567281094, 0.8265854747306287,-0.015758658987033422, -0.2873988222474053,
+                    -0.42783756578748666,  0.044081783789183565, -0.26971376257215296, 0.8615487797670971]
+
+Q_SOLUTION_ARRAY_2 = [-0.8571428571428572,   0.3942857142857143,  0.33142857142857146, 
+                      -0.4285714285714286,  -0.9028571428571428, -0.03428571428571425, 
+                       0.28571428571428575, -0.1714285714285714,   0.9428571428571428]
+
+Q_SOLUTION_ARRAY_3 = [-0.7724247413634004, -0.026670393594597247, -0.6345460653374136, 
+                      -0.5777485870360393,  -0.38541856437557026,  0.7194853024298236,
+                      -0.26375478973384403,   0.9223563413020934, 0.28229805268947933]
+
```

Comment:
`factorize_qr` in `math_spec.rb` and the lapacke methods in 'lapacke_spec.rb' share these values for their test cases. Is that okay?